cookbooks/upstage/financial_analysis.ipynb

```diff
@@ -0,0 +1,409 @@
+{
```

Comment:
<p>Please refer to other cookbooks and make consistent API key usage experience.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UpstageAI/cookbook/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>